.buildkite/scripts/common/util.sh

```diff
@@ -171,15 +171,22 @@ download_artifact() {
   retry 3 1 timeout 3m buildkite-agent artifact download "$@"
 }
 
+if [[ "$VAULT_ADDR" == *"secrets.elastic.co"* ]]; then
```

Comment:
Does this need the `TODO: remove...` comment as well so that we do not forget about it?